ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -177,6 +179,31 @@ class H2OGridSearch(override val uid: String)
       "The fit method of the grid search must be called first to be able to obtain a list of models.")
   }
 
+  private def getMetricValue(model: H2OMOJOModel, metric: H2OMetric): Double = metric match {
+    case H2OMetric.GLRMMetric =>
+      val ast = parse(model.getModelDetails())
+
+      def getColumnName(value: JValue): String = {
+        val result = for {
+          JObject(obj) <- value
+          JField("name", JString(result)) <- obj
+        } yield result
+        result.head
+      }
+
+      val metricValueOption = for {
+        JObject(obj) <- ast
+        JField("model_summary", JObject(modelSummary)) <- obj
+        JField("columns", JArray(columns)) <- modelSummary
+        ("final_objective_value", index) <- columns.arr.map(getColumnName).zipWithIndex
```

Comment:
@wendycwong I'm extracting the metric from model_summary on MOJO model. The name of the value is `final_objective_value`. Is this the value that we try to minimize during the model training?

Reply:
@mn-mikke: that is correct.